src/Datadog.Trace/RuntimeMetrics/MetricsPaths.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Datadog.Trace.RuntimeMetrics
+{
+    internal static class MetricsPaths
```

Comment:
just a nit, but "Paths" in `MetricsPaths` sounds strange to me since these are metric names.

Reply:
That's the terminology used by Graphite: https://graphite.readthedocs.io/en/latest/feeding-carbon.html#step-3-understanding-the-graphite-message-format
I guess I got used to it 😛